plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -236,5 +245,67 @@ public void testFilesTable()
+
+        MaterializedResult result = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM test_schema.\"test_table$files\" order by file_path");
+
+        MaterializedResult expectedStatistics =
+                MaterializedResult.resultBuilder(getSession(), VARCHAR, BIGINT, new MapType(INTEGER, VARCHAR, new TypeOperators()),
+                                new MapType(IntegerType.INTEGER, VARCHAR, new TypeOperators()))
+                        .row("ORC", 1L, Map.of(Integer.valueOf(1), "0", Integer.valueOf(2), "2019-09-08"), Map.of(Integer.valueOf(1), "0", Integer.valueOf(2), "2019-09-08"))
+                        .row("ORC", 2L, Map.of(Integer.valueOf(1), "1", Integer.valueOf(2), "2019-09-09"), Map.of(Integer.valueOf(1), "2", Integer.valueOf(2), "2019-09-09"))
+                        .row("ORC", 1L, Map.of(Integer.valueOf(1), "3", Integer.valueOf(2), "2019-09-09"), Map.of(Integer.valueOf(1), "3", Integer.valueOf(2), "2019-09-09"))
+                        .row("ORC", 2L, Map.of(Integer.valueOf(1), "4", Integer.valueOf(2), "2019-09-10"), Map.of(Integer.valueOf(1), "5", Integer.valueOf(2), "2019-09-10"))
+                        .build();
+
+        assertEquals(result, expectedStatistics);
+    }
+
+    @Test
+    public void testAllFilesTable()
+    {
+        assertQuery("SHOW COLUMNS FROM test_schema.\"test_table$all_files\"",
+                "VALUES ('content', 'integer', '', '')," +
+                        "('file_path', 'varchar', '', '')," +
+                        "('file_format', 'varchar', '', '')," +
+                        "('record_count', 'bigint', '', '')," +
+                        "('file_size_in_bytes', 'bigint', '', '')," +
+                        "('column_sizes', 'map(integer, bigint)', '', '')," +
+                        "('value_counts', 'map(integer, bigint)', '', '')," +
+                        "('null_value_counts', 'map(integer, bigint)', '', '')," +
+                        "('nan_value_counts', 'map(integer, bigint)', '', '')," +
+                        "('lower_bounds', 'map(integer, varchar)', '', '')," +
+                        "('upper_bounds', 'map(integer, varchar)', '', '')," +
+                        "('key_metadata', 'varbinary', '', '')," +
+                        "('split_offsets', 'array(bigint)', '', '')," +
+                        "('equality_ids', 'array(integer)', '', '')");
+        assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$all_files\"");
+
+        MaterializedResult result = computeActual("SELECT file_format, record_count, lower_bounds, upper_bounds FROM test_schema.\"test_table$all_files\" order by file_path");
+
+        MaterializedResult expectedStatistics =
+                MaterializedResult.resultBuilder(getSession(), VARCHAR, BIGINT, new MapType(INTEGER, VARCHAR, new TypeOperators()),
+                                new MapType(IntegerType.INTEGER, VARCHAR, new TypeOperators()))
```

Comment:
Drop `Integer.valueOf(1)` and use simply `1`

Same for the other usages.

Reply:
done the changes.